pkg/virt-handler/virtwrap/schema.go

```diff
@@ -29,7 +28,7 @@ func init() {
 	mapper.AddPtrConversion((**Ballooning)(nil), (**v1.Ballooning)(nil))
 	mapper.AddConversion(&Disk{}, &v1.Disk{})
 	mapper.AddConversion(&DiskSource{}, &v1.DiskSource{})
-	mapper.AddConversion(&DiskSourceHost{}, &v1.DiskSourceHost{})
+	mapper.AddPtrConversion((**DiskSourceHost)(nil), (**v1.DiskSourceHost)(nil))
```

Comment:
Ok, that did not show up, because the model.Copy test does not have the Disk attached. Could you update the test?